node/tessel-export.js

```diff
@@ -640,6 +719,29 @@ Tessel.Pin.prototype.analogWrite = function(val) {
   return this;
 };
 
+// Duty cycle should be a value between 0 and 1
+Tessel.Pin.prototype.pwmDutyCycle = function(dutyCycle, cb) {
+  // throw an error if this pin doesn't support PWM
+  if (!this.pwmSupported) {
+    throw new Error('PWM can only be used on G1 and G2 of either module port.');
+  }
+
+  if (typeof dutyCycle !== 'number' || dutyCycle > 1.0 || dutyCycle < 0) {
+    throw new Error('PWM duty cycle must be a number between 0 and 1');
+  }
+
+  // The frequency must be set prior to setting the duty cycle
+  if (Tessel.pwmPeriod === 0) {
```

Comment:
There should be some reasonable default, ideally the best period for PWM LED control.